.github/workflows/link-checker.yaml

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
```suggestion
# cspell:words lycheeverse, micalevisk
---
```

Reply:
It auto-resolves this conversation when I commit the suggestion from the website btw. Github out here trying to make me look bad!